### DISCUSSION TITLE
Is there anyway to control Corsair SP140 RGB fans?

### DISCUSSION THREAD

**Discussion by AssassinXDAhmed** [Rem0o/FanControl.Releases#1891] [Q&A]
Hi there,
I can't control the fan speeds of Corsair Fans.. I have 3 be quiet! fans and I can control them fine but the Corsair ones aren't controllable.. I exited iCUE and closed its services and still get the question mark sign when I try to control them.
The app reads the speed of the fans but it says its controlled by other software. I don't have any other software that control fan speeds other than App center by GIGABYTE and I already tried to close it but no luck.

Any suggestions?

**Comment by Rem0o** [comment]
Where are these plugged?

**Reply by AssassinXDAhmed** [reply]
One plugged into System Fan 4 and the other in System Pump 6
